esbuild.js

```diff
@@ -21,7 +21,8 @@ const clientConfig = {
     'vscode-jsonrpc',
   ],
   format: 'cjs',
-  minify: true,
+  // Do not enable minification. It seems to break the extension on windows. See #1198.
```

Comment:
ty for adding a comment, so we know to test with WSL if we ever enable minification again. 👍